### PR TITLE
Pymotif/tests

### PR DESCRIPTION
**Pull request by jasonho-lynx**
## PR Type

Tests and bugfixes for pymotif

## Detailed Description

- Tests for pymotif can be run with `pytest -v` from the test folder, see README for more details
- Some bugfixes:
  - CSV import now works with a file that just has `source` and `target` cols
  - Networkx import also works for a graph with only `source` and `target` cols
  - Neo4j import now works if there are no node labels specified in the db

## Does this PR introduce breaking change?

- No

### CODE REVIEW

File: packages/pymotif/pymotif/tests/test_motif.py [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+# coding: utf-8
+
+# Copyright (c) Cylynx.
+# Distributed under the terms of the Modified BSD License.
+
+import json
+
+from ..motif import Motif
+from .. import constants as C
+from . import constants as TC
```
Comment (timlrx) [inline packages/pymotif/pymotif/tests/test_motif.py:11]: what does `TC` stand for? test constants?

**Review by timlrx — APPROVED** [review]
Thanks for the bug fixes and tests, LGTM

File: packages/pymotif/pymotif/tests/test_motif.py (hunk 1 above)
Comment (jasonho-lynx) [reply packages/pymotif/pymotif/tests/test_motif.py:11]: Yep, shall add a comment there the next time I commit

**Comment by jasonho-lynx** [comment]
You'll deploy the updated version to PyPI?

**Comment by timlrx** [comment]
yes, along with other fixes and enhancements to Motif. I will sync it with the github releases i.e. every github release = npm release + pypi

**Pull request merged**